### ISSUE TITLE
support for quad meshes

### ISSUE THREAD

**Issue by nitinagarwal** [jpcy/xatlas#119]
Does xatlas support unwrapping and packing of quad meshes?
Is that on the roadmap?

**Comment by rondreas** [comment]
It is supported, when you make the xatlas::MeshDecl give it a vector of int for number of vertices per face,vector should be size of face count

https://github.com/jpcy/xatlas/blob/f700c7790aaa030e794b52ba7791a05c085faf0c/source/xatlas/xatlas.h#L124-L127

**Comment by nitinagarwal** [comment]
Thanks for your reply. 
Do you also support quad meshes which already have uvs. 
A potential use case could be to just use the xatlas packing algorithm with predefined uvs. 

I couldnt find it in https://github.com/jpcy/xatlas/blob/f700c7790aaa030e794b52ba7791a05c085faf0c/source/xatlas/xatlas.h#L156

**Comment by mifth** [comment]
Yeah, it would be great to have quads for uv meshes, As a workaround we can triangulate UVs/polygons manually.

**Comment by mifth** [comment]
There were some issues with Quads and NGons. I've fixed them in this pull request:
https://github.com/jpcy/xatlas/pull/138
